SampleApp/Components/SwitchView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import Warp
+
+struct SwitchView: View {
+    
+    @State private var isOn = true
+    @State private var switchState: Warp.SwitchState = .default
+    
+    var body: some View {
+        ScrollView(showsIndicators: false) {
+            // Switch display with dynamic state
+            Warp.Switch(
+                isOn: $isOn,
+                state: switchState
+            )
+            .padding()
+            
+            // Controls to modify Switch's state
+            GroupBox(content: {
+                VStack(alignment: .leading) {
+                    // Toggle Switch State
+                    HStack {
+                        Text("Toggle Switch State")
+                        Spacer()
+                        
+                        Warp.Button.create(
```

Comment:
Why is it a button, can't we use the Warp.Switch here?

Reply:
Warp.Switch(isOn: $isDisabled)

Or maybe a segment

Picker("Enable or Disable", selection: $isDisabled) {
    Text("Enable").tag(false)
    Text("Disable").tag(true)
}
.pickerStyle(SegmentedPickerStyle())